src/analyses/goto_check.cpp

```diff
@@ -1747,6 +1749,26 @@ void goto_checkt::check_rec(const exprt &expr, guardt &guard)
     if(check_rec_member(to_member_expr(expr), guard))
       return;
   }
+  else if(expr.id() == ID_forall || expr.id() == ID_exists)
+  {
+    // In the case of a quantified statement, skolemise the expression
+    // and recurse.
+    const auto quantified_expr = to_quantifier_expr(expr);
+    const auto quantifier_sym = quantified_expr.symbol();
```

Comment:
Clang-tidy suggests this should be `const auto &` to avoid unnecessary copy construction.

Reply:
No longer relevant.